lib/ews/connection.rb

```diff
@@ -195,26 +220,32 @@ def check_response(resp, include_http_headers: false, request_body: nil)
 
     case resp.status
     when 200
+      log msg: "Recieved a 200 response for request of type: **#{options[:request_type]}** with uniq id: **#{options[:uniq_id]}** and headers of: #{resp.headers}"
```

Comment:
shall we use just `#{resp.status}` in the all messages and move it few lines up to a more general message?